### ISSUE TITLE
ResourceLabels is missing in Go SDK

### ISSUE THREAD

**Issue by yarinm** [pulumi/pulumi-gcp#958]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

The Go SDK is missing the property `resourceLabels` for `NodePoolNodeConfigArgs` struct

### Affected area/feature

Unable to set this property

**Comment by tal-lich** [comment]
It's important to note that labels are commonly used for cost tracking, being unable to set them prevents this altogether.

**Comment by squaremo** [comment]
Yes indeed, I think it is worth updating to the latest version of the SDK (6.46.0) -- I believe this is what you're looking for @yarinm: https://www.pulumi.com/registry/packages/gcp/api-docs/container/nodepool/#labels_go
Thanks for looking into this @tal-lich, good digging!
